src/anomalib/utils/metrics/anomaly_score_threshold.py

```diff
@@ -60,3 +60,51 @@ def compute(self) -> Tensor:
         else:
             self.value = thresholds[torch.argmax(f1_score)]
         return self.value
+
+
+class MeanVarianceMode(str, Enum):
+    """Supported Mean Variance Combination Type"""
+
+    ADD = "add"
+    MINUS = "minus"
+
+
+class AnomalyScoreMeanVarThreshold(AnomalyScoreThreshold):
+    """Anomaly Score Threshold.
+
+    This class computes/stores the threshold that determines the anomalous label
+    given anomaly scores. The computation is based on the mean and variance.
+
+    """
+
+    STD_DEV_RATIO = 0.1
+    MEANVAR_MODE = MeanVarianceMode.ADD
```

Comment:
Is it possible to avoid global variables? Can we add it to the `__init__` args?

Reply:
fixed.